City-of-Helsinki/helsinki-paatokset
# PP-256: Decisionmaker fixes for trustees and officals

**To test**
- Checkout branch, run `make new` (take database backup first)
- Login to container and run `drush ap:fs -v;drush mim ahjo_decisionmakers:all`
- This should create decision makers without any errors
- Open this node: https://helsinki-paatokset.docker.so/fi/paattajat/sosiaali-ja-terveyslautakunnan-jaosto
  - The subnavigation links should now work and they should use the correct last part from the URL
  - Edit the node and change the title to something
  - The URL of the page should not change
  - The Ahjo Title should appear in the tag below the main title
- Run the migration again: `drush mim ahjo_decisionmakers:all --update`
  - The title you edited should still be there
- Open this node: https://helsinki-paatokset.docker.so/fi/paattajat/apulaispormestari-2
  - There should be a subheading below the main title, coming from the "Organization name" field. This should match the name in the Organization Level Above JSON data.
  - Check that the "decisions" link works, and takes you to https://helsinki-paatokset.docker.so/fi/paattajat/apulaispormestari-2/decisions and not https://helsinki-paatokset.docker.so/fi/paattajat/apulaispormestari/decisions
